Dockerfile.worker

```diff
@@ -23,5 +23,6 @@ RUN cd $PS_PATH && \
     ansible-playbook -vv -c local -i localhost, recipe-worker.yaml
 
 COPY . $PS_PATH
+WORKDIR $PS_PATH
```

Comment:
Sorry for being late for the party, but, can you help me understand how this works?
I'm asking because I was going the opposite way and [removed the `$PS_PATH`](https://github.com/packit-service/packit-service/pull/167/commits/167b75cf7cde56795eae74b65fd2bccca6431181).
I see the comment in `Dockerfile.tests` saying that it's "so that we don't need to rerun the playbook above for every code change", but I still don't get it. Can't you volume mount the dir where the code is installed (currently you volume mount the dir from which the code is installed, don't you?)

Reply:
Sadly you can't use hostDir mount in `oc cluster up`: https://github.com/packit-service/packit-service/pull/171/files#diff-b67911656ef5d18c4ae36cb6741b7965R64

we can comment it out by default, or I can write more docs/comments

Reply:
Ok, I'm fine with that, even I don't know what the hostDir mount is and why it works for `$PS_PATH`, but not for the `/usr/local/lib/python3.7/site-packages/packit_service`. Don't we need this in Dockerfile.worker.prod too?